packages/rest-api/src/tests/getBridgeLimits.test.ts

```diff
@@ -0,0 +1,81 @@
+import request from 'supertest'
+import express from 'express'
+
```

Comment:
rename this test file to `bridgeLimitsRoute.test.ts` to match the rest of the tests. we might write controller/middleware tests later, so this naming convention keeps them unique.